src/Rule/Date.php

```diff
@@ -173,7 +174,9 @@ class Date extends Rule
      * If this property is null, the value of [[min]] will be used (before parsing).
      */
     public $minString;
```

Comment:
We need to make these non-public. Same for all other properties.

Reply:
Are we using `_` prefix for private properties?

Reply:
No.